Sources/Crypto/CryptoError_boring.swift

```diff
@@ -0,0 +1,4 @@
+@available(iOS 13.0, macOS 10.15, watchOS 6.0, tvOS 13.0, *)
```

Comment:
Sorry to be a bore here, but you'll need the license header on this file.

Reply:
Ah good point, added